msal/application.py

```diff
@@ -549,6 +544,9 @@ def __init__(
                     )
             else:
                 raise
+
+        if allow_broker and sys.platform != "win32":
```

Comment:
What if I set `allow_broker` on Windows 7? This won't trigger error.

Reply:
On Windows 7, it will indeed proceed and [silently fallback to browser](https://github.com/AzureAD/microsoft-authentication-library-for-python/blob/1.20.0/msal/application.py#L563-L566). This is considered OK, because we foresee no broker for Windows 7.